### PR TITLE
Design tweaks

### LINKED ISSUE #262: Filter in StopVC crashes app
Console
```
Assertion failed: Duplicate tag sortTag: file /Users/alanchu/Library/Developer/Xcode/DerivedData/OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/BaseRow.swift, line 198
2020-07-23 23:32:04.533536-0700 App[3501:636922] Assertion failed: Duplicate tag sortTag: file /Users/alanchu/Library/Developer/Xcode/DerivedData/OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/BaseRow.swift, line 198
```

Stack Trace
```
#3	0x00000001b1a6acb8 in _assertionFailure(_:_:file:line:flags:) ()
#4	0x00000001059441fc in BaseRow.wasAddedTo(section:) at .../OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/BaseRow.swift:198
#5	0x00000001059ac154 in Section.wasAddedTo(form:) at .../OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/Section.swift:382
#6	0x0000000105976ad8 in Form.append(_:) at .../OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/Form.swift:205
#7	0x000000010598fae8 in +++ infix(_:_:) at .../OBAKit-crmfsejwmraydyccaehdchkxylrw/SourcePackages/checkouts/Eureka/Source/Core/Operators.swift:51
#8	0x00000001058bdca8 in StopPreferencesViewController.viewDidLoad() at .../OBAKit/OBAKit/Stops/StopPreferencesViewController.swift:43
```

### PR DESCRIPTION
**Pull request by ualch9**
A handful of design tweaks,
- Updates icons to SF symbols
- Make table header text larger
- Implement LoadMoreButton from #202
- Fix #262

|Before|After|
|---|---|
|||

### CODE REVIEW

**Comment by aaronbrethorst** [comment]
We can't extract and ship icons from SF Symbols. We basically can't use it until the app supports 13.x and up.

**Comment by ualch9** [comment]
@aaronbrethorst Extracting from SFSymbols.app and using it in any version of iOS appears to be OK, according to this post in the apple dev forums,
> If you are using the symbol (or the derived work from it) generated with the SF Symbols app in and application running in Apple’s iOS, iPadOS, macOS and tvOS operating systems, then it is allowed. Your request is specifically about your app running in iOS <13 so it qualifies. However, you cannot use the symbol or derived work in another platform, web application, etc. as your app is/may no longer be running in the operating systems previously mentioned.
> 
> Note however that you lose a lot of functionality when using a pdf/png in iOS <13. The system won’t automatically align your image, use the appropriate scale, adapt to bold text or dynamic type. You will have to do all of this manually. Finally, all symbols were re-designed for iOS 13. Depending on the use, it may not fit very well with the rest of the OS.
> Posted 4 weeks ago by Frameworks Engineer 

https://developer.apple.com/forums/thread/131744?answerId=615428022#615428022

**Comment by ualch9** [comment]
drafting to improve implementation...

**Comment by ualch9** [comment]
@aaronbrethorst ok for review

File: OBAKit/Theme/Icons.swift [hunk 1]
```diff
@@ -14,25 +14,44 @@ import OBAKitCore
...
+
+    /// The Bookmarks tab selected icon, for apps using a tab bar UI metaphor.
+    public class var bookmarksSelectedTabIcon: UIImage {
+        configureForTabIcon(systemImage(named: "bookmark.fill", fallback: "bookmark_selected"))
     }
 
     /// A More tab icon, for apps using a tab bar UI metaphor.
     public class var moreTabIcon: UIImage {
-        imageNamed("more")
+        configureForTabIcon(systemImage(named: "ellipsis.circle", fallback: "more"))
```
Comment (aaronbrethorst) [inline OBAKit/Theme/Icons.swift:49]: please change to `systemImage(named: "ellipsis", fallback: "more")` and drop the `moreSelectedTabIcon`

**Comment by aaronbrethorst** [comment]
Also, I'm happy Apple clarified the SF Symbols/iOS 12 thing. Nice work digging that up!

File: OBAKit/Theme/Icons.swift (hunk 1 above)
Comment (ualch9) [reply OBAKit/Theme/Icons.swift:49]: This is a workaround for an alignment issue with `ellipsis`.
Simulator Screen Shot - iPhone 11 Pro - 2020-07-27 at 23 21 13

**Pull request merged**